### DISCUSSION TITLE
Why does the Sign Up button have a `disabled` attribute in the Command Log even though it's clickable?

### DISCUSSION THREAD

**Discussion by mccataldo** [cypress-io/cypress-realworld-app#735] [General]
As you can see in the screenshot below, I'm able to click the Sign Up button and the button isn't disabled in the DOM snapshot, but according to the Command Log output, the `button` element has the `disabled` attribute. Why is that?

If you want to reproduce it, check out [this commit](https://github.com/iheartjs-dev/cypress-realworld-app/commit/629ab6aab954caa7aabca3d84e20697da14463c1) to my RWA fork.

![Screen Shot 2020-10-07 at 10 49 42 AM](https://user-images.githubusercontent.com/14354013/95362265-14f4dc80-088b-11eb-9ff1-9f000c053090.png)

Update: 
Adding assertions didn't affect the results, but FWIW, I did notice that the element in the second assertion message doesn't have the `Mui-disabled` class while the element output to the Console does.
![Screen Shot 2020-10-07 at 11 02 01 AM](https://user-images.githubusercontent.com/14354013/95363321-8aad7800-088c-11eb-91dc-3f0493f3292e.png)

**Comment by kevinold** [comment]
@mccataldo The `disabled` attribute is controlled via the Material UI prop based on if the form is valid or if it's being submitted.    See https://github.com/cypress-io/cypress-realworld-app/blob/develop/src/components/SignUpForm.tsx#L183
